astropy/time/core.py

```diff
@@ -3179,6 +3179,8 @@ def to(self, unit, equivalencies=[]):
         --------
         to_value : get the numerical value in a given unit.
         """
+        if equivalencies is None:
+            equivalencies = []
```

Comment:
No, this is wrong: just like for `units`, `equivalencies=None` has concrete meaning, so this changes API.